Makefile

```diff
@@ -1,4 +1,7 @@
 # Run jekyll in development mode
+
+LOCAL_DOCS_DIR := docs/local
```

Comment:
Nitpick: did we already consider making this configurable? I would recommend making it configurable by doing `LOCAL_DOCS_DIR ?= docs/local` so we can override it at runtime if desired:

```
make LOCAL_DOCS_DIR=docs/myversion run_docs_local
```